packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1241,6 +1251,56 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     }
   }
 
+  private _notifyPreviewChanged(prevState: IComboBoxState) {
+    const { onPreviewExecute, onRevertPreviewExecute } = this.props;
+    const {
+      currentPendingValue,
+      currentOptions,
+      currentPendingValueValidIndex,
+      currentPendingValueValidIndexOnHover
+    } = this.state;
+
+    if (onPreviewExecute && onRevertPreviewExecute) {
```

Comment:
Consider pulling this if check to the top of the function negating it and returning if it's true so the whole function isn't nested inside of the check